src/keysend/tests.rs

```diff
@@ -8,10 +8,12 @@ async fn test_keysend() {
     let zbdenv: String =
         env::var("ZBD_ENV").unwrap_or_else(|_| String::from("https://api.zebedee.io"));
     let zebedee_client = ZebedeeClient::new().domain(zbdenv).apikey(apikey).build();
+    let tlvs = vec![TlvRecord { record_type : 123123123, value : String::from("00ABCDEF") }];
 
     let keysend_payload = Keysend {
         amount: String::from("1000"),
-        pubkey: String::from("0332d57355d673e217238ce3e4be8491aa6b2a13f95494133ee243e57df1653ace"),
+        pubkey: String::from("03f80288f858251aed6f70142fab79dede5427a0ff4b618707bd0a616527a8cec7"),
```

Comment:
lol

Reply:
Probably should be an ENV variable like `ZBD_API_KEY` is.